### ISSUE TITLE
Table Cell styling not working properly

### ISSUE THREAD

**Issue by Grafikart** [wkhtmltopdf/wkhtmltopdf#2469]
I have a weird bug when trying to use the option --user-style-sheet.

## Problem

I have the following html / css

    HTML
    <html><body><table><tr><td>Je fais un test</td></tr></table></body></html>

    CSS
    td{ border: 2px solid #FF0000; color: #FF0000; }

If I import the css with the `<link> ` element, wkhtmltopdf will print the border and the color of my td
If I use `--user-style-sheet` option, wkhtmltopdf will only render the color, and not the 

Basically if a stylesheet is imported using `--user-style-sheet` padding and borders for table cells seems to be ignored.

## Temporary fix 

Don't use `--user-style-sheet` and use absolute path with the  `<link> `  element. This issue could be related to #1920

**Comment by imuller** [comment]
@Grafikart Thanks, your temporary fix solved the issue for me to. Don't know if <link> is slower for generating the pdf-document, or maybe the same?

**Comment by Tomsgu** [comment]
Is this still an issue in 0.12.5?

**Comment by MattApril** [comment]
Seems to be so. I just had the same issue and solved it with the same solution.

**Comment by dlondero** [comment]
Had the same issue with `td` borders not being printed. Fallback to including stylesheet in the layout of the generated html, works like a charm.

**Comment by ashkulz** [comment]
@dlondero: can you check if it goes away with the 0.13 builds?

**Comment by dlondero** [comment]
@ashkulz I can try. Where can I find the package for Debian9?

**Comment by ashkulz** [comment]
There isn't one yet, maybe try the Debian 8 image?
